Configuration/StandardSequences/python/ctppsRecoForSimulation2017_cff.py

```diff
@@ -0,0 +1,272 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this looks like a temporary test config. it doesn't belong in Configuration/StandardSequences main python dir.
please remove.